DataConnectors/Forcepoint CASB.json

```diff
@@ -0,0 +1,111 @@
+{
+    "id": "Forcepoint",
+    "title": "Forcepoint CASB",
+    "publisher": "Forcepoint CASB",
+    "descriptionMarkdown": "The Forcepoint CASB (Cloud Access Security Broker) Connector allows you to automatically export CASB logs and events into Azure Sentinel in real-time. This enriches visibility into user activities across locations and cloud applications, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel​",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CommonSecurityLog",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint CASB\"\n"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 5 Users With The Highest Number Of Logs",
+            "query": "CommonSecurityLog \n| summarize Count = count() by DestinationUserName\n| top 5 by DestinationUserName\n| render barchart"
+        },
+      {
+            "description" : "Top 5 Users by Number of Failed Attempts ",
+            "query": "CommonSecurityLog \n| extend outcome = split(split(AdditionalExtensions, \";\", 2)[0], \"=\", 1)[0]\n| extend reason = split(split(AdditionalExtensions, \";\", 3)[0], \"=\", 1)[0]\n| where outcome ==\"Failure\"\n| summarize Count= count() by DestinationUserName\n| render barchart"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (CEF)",
```

Comment:
Update "name": "CommonSecurityLog (CEF)", to
 "name": "CommonSecurityLog (ForcepointCASB)",